lib/inc/internal/facts/linux/os_cisco.hpp

```diff
@@ -57,7 +57,17 @@ namespace facter { namespace facts { namespace linux {
                 return value;
             }
             auto val = _release_info.find("ID_LIKE");
-            return (val != _release_info.end()) ? val->second : std::string();
+            if (val != _release_info.end()) {
+                auto family = val->second;
+                auto val2 = family.find(" ");
```

Comment:
Another nit pick: this variable isn't a value, but a position into the string, so it's better called `position`, `pos`, `index` etc imo.